NEWS

```diff
@@ -1,6 +1,12 @@
 # bedr 1.0.7
```

Comment:
I think the version should be `1.1.0` (i.e. new feature) and DESCRIPTION should be updated as well. @whelena 

https://r-pkgs.org/release.html#release-version

@SyedHaider5 @dan-knight any concerns with CRAN submission?

Reply:
Do I need to add my name as contributors?

Reply:
Yes, please!